### PR TITLE
Split V2 App

### PR DESCRIPTION
**Pull request by shivani-sumo**
## PLEASE READ

Following a recent back-end update, all contributors with a local clone or fork of our repository are required to run `yarn install`. This does not apply to [direct page edits](https://help.sumologic.com/docs/contributing/edit-doc/#minor-edits).

- [ ] Yes, I've run `yarn install`
- [ ] No, does not apply to me

## Purpose of this pull request

This pull request...

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/integrations/webhooks/split.md [hunk 1]
```diff
@@ -8,15 +8,94 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 
 <img src={useBaseUrl('img/integrations/webhooks/split-logo.png')} alt="Thumbnail icon" width="50"/>
 
+The Split App for Sumo Logic enables you to seamlessly monitor feature flagging, experiment results, and user behavior, enabling data-driven decision-making and fostering a more agile and competitive development process. This app is based on Split Webhook, which provides seamless integration between Split and Sumo Logic.
```
Comment (JV0812) [inline docs/integrations/webhooks/split.md:11]:
```suggestion
The Split app for Sumo Logic enables you to seamlessly monitor feature flagging, experiment results, and user behaviour, enabling data-driven decision-making and fostering a more agile and competitive development process. This app is based on Split Webhook, which provides seamless integration between Split and Sumo Logic.
```

File: docs/integrations/webhooks/split.md [hunk 2]
```diff
@@ -8,15 +8,94 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
+
 Split is a feature delivery platform that combines the quick and dependable nature of feature flags with data-driven insights to assess the effects of each feature. You can use a webhook in the Split platform to forward activities related events to the Sumo Logic HTTP endpoint. Using these logs, you can monitor user activities, admin changes in the tools used by the whole team, and impressions in Sumo Logic. For more details, refer to the [Split Documentation](https://docs.split.io/docs).
 
 ## Event types
 
-The Sumo Logic integration for Split ingests Split events into Sumo Logic through an outgoing webhook available in Split. The following event types are ingested through the Split webhook:
+The Sumo Logic App for Split ingests Split events into Sumo Logic through an outgoing webhook available in Split. The following event types are ingested through the Split webhook:
```
Comment (JV0812) [inline docs/integrations/webhooks/split.md:17]:
```suggestion
The Sumo Logic app for Split ingests Split events into Sumo Logic through an outgoing webhook available in the Split. The following event types are ingested through the Split webhook:
```

File: docs/integrations/webhooks/split.md [hunk 3]
```diff
@@ -8,15 +8,94 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
 - Audit logs
 - Admin audit logs
 - Impressions
 
+## Log Types
```
Comment (JV0812) [inline docs/integrations/webhooks/split.md:22]:
```suggestion
## Log types
```

File: docs/integrations/webhooks/split.md [hunk 4]
```diff
@@ -8,15 +8,94 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
+
+### Sample Log Message
```
Comment (JV0812) [inline docs/integrations/webhooks/split.md:24]:
```suggestion
### Sample log message
```

File: docs/integrations/webhooks/split.md [hunk 5]
```diff
@@ -8,15 +8,94 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
+      "name": {
+         "from": null,
+         "to": "client-side - 7645a - 7653b"
+      }
+   },
+   "type": "audit_log"
+}
+```
+
+### Sample Query
```
Comment (JV0812) [inline docs/integrations/webhooks/split.md:88]:
```suggestion
### Sample query
```

File: docs/integrations/webhooks/split.md [hunk 6]
```diff
@@ -53,3 +132,42 @@ Follow the below steps to configure the Split webhook.
 - For detailed information about webhook creation, refer to the [Split Webhook - audit log](https://help.split.io/hc/en-us/articles/360020957991-Webhook-audit-log), [Split Webhook - admin audit logs](https://help.split.io/hc/en-us/articles/360051384832-Webhook-admin-audit-logs) and [Split Webhook - impressions](https://help.split.io/hc/en-us/articles/360020700232-Webhook-impressions) documentation.
 - For support, contact [Split](https://www.split.io/support/).
 :::
+
+### Installing the App
```
Comment (JV0812) [inline docs/integrations/webhooks/split.md:136]:
```suggestion
### Installing the Split app
```

File: docs/integrations/webhooks/split.md [hunk 7]
```diff
@@ -53,3 +132,42 @@ Follow the below steps to configure the Split webhook.
...
+    1. **Key.** Select either of these options for the data source.
+        * Choose **Source Category**, and select a source category from the list for **Default Value**.
+        * Choose **Custom**, and enter a custom metadata field beginning with an underscore. Insert it's value in **Default Value**.
+5. Click on **Next**. You will be redirected to **Preview & Done** section.
+
+Given App will be installed in **Installed Apps** folder and Panels of dashboard will start to fill automatically.
+
+Each panel slowly fills with data matching the time range query and received since the panel was created. Results will not immediately be available, updating with full graphs and charts over time.
+
+## Split Dashboards
```
Comment (JV0812) [inline docs/integrations/webhooks/split.md:155]:
```suggestion
## Viewing Split dashboards
```

File: docs/integrations/webhooks/split.md [hunk 8]
```diff
@@ -53,3 +132,42 @@ Follow the below steps to configure the Split webhook.
...
+
+### Overview
+
+**Split - Overview** dashboard offers transparency into actions performed by both administrators and team members, delivering valuable insights into audit events, their distribution, and statistics categorized by their respective types.
```
Comment (JV0812) [inline docs/integrations/webhooks/split.md:159]:
```suggestion
The **Split - Overview** dashboard offers transparency into actions performed by both administrators and team members, delivering valuable insights into audit events, their distribution, and statistics categorized by their respective types.
```

File: docs/integrations/webhooks/split.md [hunk 9]
```diff
@@ -53,3 +132,42 @@ Follow the below steps to configure the Split webhook.
...
+
+<img src={useBaseUrl('img/integrations/webhooks/Split-Overview.png')} alt="Split-Overview" />
+static/img/integrations/webhooks/Split-Overview.png
```
Comment (JV0812) [inline docs/integrations/webhooks/split.md:162]:
```suggestion
```

File: docs/integrations/webhooks/split.md [hunk 10]
```diff
@@ -53,3 +132,42 @@ Follow the below steps to configure the Split webhook.
...
+
+Each panel slowly fills with data matching the time range query and received since the panel was created. Results will not immediately be available, updating with full graphs and charts over time.
+
+## Split Dashboards
+
+### Overview
+
+**Split - Overview** dashboard offers transparency into actions performed by both administrators and team members, delivering valuable insights into audit events, their distribution, and statistics categorized by their respective types.
+
+<img src={useBaseUrl('img/integrations/webhooks/Split-Overview.png')} alt="Split-Overview" />
```
Comment (JV0812) [inline docs/integrations/webhooks/split.md:161]:
```suggestion
<img src={useBaseUrl('img/integrations/webhooks/Split-Overview.png')} style={{border: '1px solid black'}} alt="Split-Overview" />
```

File: docs/integrations/webhooks/split.md [hunk 11]
```diff
@@ -53,3 +132,42 @@ Follow the below steps to configure the Split webhook.
...
+static/img/integrations/webhooks/Split-Overview.png
+
+### Users and Groups
+
+**Split - Users and Groups** dashboard offers concise statistical summaries pertaining to Split users and groups, including administrative actions taken concerning them.
```
Comment (JV0812) [inline docs/integrations/webhooks/split.md:166]:
```suggestion
The **Split - Users and Groups** dashboard offers concise statistical summaries pertaining to Split users and groups, including administrative actions taken concerning them.
```

File: docs/integrations/webhooks/split.md [hunk 12]
```diff
@@ -53,3 +132,42 @@ Follow the below steps to configure the Split webhook.
...
+
+<img src={useBaseUrl('img/integrations/webhooks/Split-Users_and_Groups.png')} alt="Split-Users_and_Groups" />
```
Comment (JV0812) [inline docs/integrations/webhooks/split.md:168]:
```suggestion
<img src={useBaseUrl('img/integrations/webhooks/Split-Users_and_Groups.png')} style={{border: '1px solid black'}} alt="Split-Users_and_Groups" />
```

**Review by JV0812 — CHANGES_REQUESTED** [review]
Great work @shivani-sumo. I have some minor comments. Also, can you please crop the images and remove the extra space.

File: docs/integrations/webhooks/split.md (hunk 1 above)
Comment (kimsauce) [inline docs/integrations/webhooks/split.md:11]:
```suggestion
The Split App for Sumo Logic enables you to seamlessly monitor feature flagging, experiment results, and user behavior, enabling data-driven decision making and fostering a more agile and competitive development process. This app is based on Split Webhook, which provides seamless integration between Split and Sumo Logic.
```

**Review by JV0812 — APPROVED** [review]
LGTM

File: docs/integrations/webhooks/postman.md [hunk 13]
```diff
@@ -55,3 +121,31 @@ Follow the below steps to configure the Postman webhook.
 - For detailed information about webhook creation, refer to the [Postman Documentation](https://learning.postman.com/docs/integrations/webhooks/).
 - For support, [contact Postman](https://support.postman.com/hc/en-us).
 :::
+
+### Installing the Postman app
+
+This section has instructions for installing the Sumo Logic app for Postman.
```
Comment (JV0812) [inline docs/integrations/webhooks/postman.md:127]:
```suggestion
```

File: docs/integrations/webhooks/sentry.md [hunk 14]
```diff
@@ -56,3 +556,25 @@ Follow the below steps to configure the Sentry Webhook.
 - For detailed information about webhook creation, refer to the [Sentry Documentation](https://docs.sentry.io/product/integrations/integration-platform/webhooks/).
 - For support, [contact Sentry](https://help.sentry.io/).
 :::
+
+### Installing the Sentry app
+
+This section has instructions for installing the Sumo Logic app for Sentry.
```
Comment (JV0812) [inline docs/integrations/webhooks/sentry.md:562]:
```suggestion
```

File: docs/integrations/webhooks/split.md [hunk 15]
```diff
@@ -53,3 +132,25 @@ Follow the below steps to configure the Split webhook.
 - For detailed information about webhook creation, refer to the [Split Webhook - audit log](https://help.split.io/hc/en-us/articles/360020957991-Webhook-audit-log), [Split Webhook - admin audit logs](https://help.split.io/hc/en-us/articles/360051384832-Webhook-admin-audit-logs) and [Split Webhook - impressions](https://help.split.io/hc/en-us/articles/360020700232-Webhook-impressions) documentation.
 - For support, contact [Split](https://www.split.io/support/).
 :::
+
+### Installing the Split app
+
+This section has instructions for installing the Sumo Logic app for Split.
```
Comment (JV0812) [inline docs/integrations/webhooks/split.md:138]:
```suggestion
```

File: docs/integrations/webhooks/squadcast.md [hunk 16]
```diff
@@ -82,3 +197,25 @@ Follow the below steps to configure the Squadcast webhook.
 - For detailed information about webhook creation, refer to the [Squadcast Documentation](https://support.squadcast.com/integrations/outgoing-webhooks).
 - For support, [contact Squadcast](https://www.squadcast.com/support-ticket-form).
 :::
+
+### Installing the Squadcast app
+
+This section has instructions for installing the Sumo Logic app for Squadcast.
```
Comment (JV0812) [inline docs/integrations/webhooks/squadcast.md:203]:
```suggestion
```

File: docs/integrations/webhooks/stripe.md [hunk 17]
```diff
@@ -52,3 +103,31 @@ Follow the below steps to configure the Stripe webhook.
 - For detailed information about webhook creation, refer to the [Stripe Documentation](https://stripe.com/docs/webhooks).
 - For support, [contact Stripe](https://support.stripe.com/contact/email).
 :::
+
+### Installing the Stripe app
+
+This section has instructions for installing the Sumo Logic app for Stripe.
```
Comment (JV0812) [inline docs/integrations/webhooks/stripe.md:109]:
```suggestion
```

**Comment by JV0812** [comment]
The intent of this PR is covered in the PR https://github.com/SumoLogic/sumologic-documentation/pull/3437.

**Pull request closed without merging**